bigquery/google/cloud/bigquery/magics.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2018 Google LLC
```

Comment:
No need to update copyright headers. It should indicate when the file was created, not necessarily when it was last updated.

Reply:
I see. Will revert.